config/routes.rb

```diff
@@ -14,4 +14,5 @@
   resources :users
   resources :account_activations, only: [:edit]
```

Comment:
nếu chỉ dùng 1 action của resources, em viết ngắn lại như này nhé:
```suggestion
  resources :account_activations, only: :edit
```